lib/timber/logger_backends/http.ex

```diff
@@ -382,38 +385,52 @@ defmodule Timber.LoggerBackends.HTTP do
     Logger.compare_levels(lvl, min) != :lt
   end
 
-  defp run_http_preflight_check!(http_client, api_key) do
-    auth_token = Base.encode64(api_key)
-    preflight_url = Config.preflight_url()
-
-    headers = %{
-      "Authorization" => "Basic #{auth_token}"
-    }
-
-    case http_client.request(:get, preflight_url, headers, "") do
-      {:ok, status, _, _} when status in 200..299 ->
-        :ok
-      {:ok, status, _, _} ->
-        raise TimberAPIKeyInvalid, api_key: api_key, status: status
-      _ ->
-        raise TimberAPIKeyInvalid, api_key: api_key
-    end
+  @spec handle_hackney_response({:hackney_response, reference, term}, t) :: {:ok, t}
+  # Handles responses from Hackney asynchronous requests and modifies the state appropriately
+  #
+  # This function assumes that its caller has already matched the reference being given to
+  # the one existing in the state
+  defp handle_hackney_response({:hackney_response, ref, {:ok, 401, reason}}, state) do
+    Timber.debug fn -> "HTTP request #{inspect(ref)} received response 401 #{reason}" end
+
+    raise TimberAPIKeyInvalid, status: 401, api_key: state.api_key
   end
 
-  #
-  # Errors
-  #
+  defp handle_hackney_response({:hackney_response, ref, {:ok, 403, reason}}, state) do
+    Timber.debug fn -> "HTTP request #{inspect(ref)} received response 403 #{reason}" end
 
-  defmodule NoHTTPClientError do
-    defexception message: \
-      """
-      An HTTP client could not be found. Timber allows you to choose your HTTP
-      client, but comes with a default :hackney click. Please use this via:
+    {:ok, state}
+  end
 
-        config :timber, http_client: Timber.Transports.HTTP.HackneyClient
-      """
+  defp handle_hackney_response({:hackney_response, ref, {:ok, status, reason}}, state) do
```

Comment:
Correct. Essentially, so long as it's not 401 we log it and move on. 403 is specifically called out for future use, though it doesn't have to be at this point. The backend just won't report 403 at the moment.